docs/BreakingChanges/141 WPF TreeViewItem must be used within a TreeView.md

```diff
@@ -0,0 +1,27 @@
+## ID: WPF TreeViewItem must be used within a TreeView
+
+### Scope
+Major
```

Comment:
We can give a 'version fixed' property which indicates the break is fixed in a given version (4.6.2, in this case). And we can call out the hotfix availability as a possible mitigation in 'recommended changes'